containers/podman-desktop
# fix: error text on podman commands

### What does this PR do?
It uses xterm.js to render podman CLI errpors

### Screenshot/screencast of this PR
<img width="1163" alt="Screenshot 2022-11-29 at 16 43 04" src="https://user-images.githubusercontent.com/929743/204560285-03c07803-f88d-4983-a494-69207375d017.png">




### What issues does this PR fix or reference?
Fix https://github.com/containers/podman-desktop/issues/809


### How to test this PR?
Try to initialise podman from UI, in case if podman cannot create machine, you should see a readable error message.
